IS7024/Pages/HollywoodBowl.cshtml

```diff
@@ -41,29 +45,18 @@
                     var newhour = Convert.ToInt32(mhour);
                     int realhour = newhour - 12;
                     string starttime = realhour + ":" + mmin + " PM";
-
```

Comment:
Removing space. You will see this type of edit throughout these changes. In order to not fill this up with too many comments, I will not highlight any further space edits.